bench/db_bench.ml

```diff
@@ -17,7 +17,7 @@ let hash_size = 30
 
 let value_size = 13
 
-let nb_entries = 10_000_000
+let nb_entries = 100_000
```

Comment:
Why do you change that variable in this PR?

Reply:
By mistake, fixed.